.github/workflows/codeql.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         id: always
         run: |
           if [ "${{ github.event_name }}" = "pull_request" ]; then
-            if [ "${{ contains(github.event.pull_request.labels.*.name, 'ci/codeql') }}" = "true" ]; then
+            if [ "${{ contains(github.event.pull_request.labels.*.name, 'run-ci/codeql') }}" = "true" ]; then
```

Comment:
Should it be `==`? 

Also, I believe you don't need `== "true"` (or `== "false"`) when using functions that return boolean.

Reply:
Nope, this is shell script, `==` is only usable with `[[ ]]` syntax and has no advantage here (and thus is not used simply to make this a bit more compact), and the substitution produces a string we have to match on.